docs/application/dotnet/guides/nui/imageview.md

```diff
@@ -1,53 +1,117 @@
+
+2.  Use `PixelArea` property to apply the zoom effect:
+
+    ```csharp
+    imageView.PixelArea = new RelativeVector4(0.25f, 0.25f, 0.5f, 0.5f);
     ```
 
-<a name="properties"></a>
-## Image View Properties
+3.  To rotate the image by 45 degrees, set the `Orientation` property as follows:
+
+    ```csharp
+    imageView.Orientation = new Rotation(new Radian((float)System.Math.PI / 4), 
+                                         new Vector3(0.0f, 0.0f, 1.0f));
+    ```
+
+## ImageView Event
+
+After all the required resources are loaded and ready, you can perform some actions:
+
+1. Create handler:
+
+    ```csharp
+    public void onResourceReady(object sender, ImageView.ResourceReadyEventArgs e)
+    {
+        // Resource is ready, do something
+    } 
+    ```
+
+2. Add handler to `ImageView` by using the `ResourceReady` property:
+
+    ```csharp
+    imageView.ResourceReady += onResourceReady;
+    ```
+
+## ImageView Properties
+
 
-The following table defines the [Tizen.NUI.BaseComponents.ImageView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.ImageView.html) class control properties.
+The following table defines the `ImageView` class control properties.
 
-**Table: Image view control properties**
+**Table: ImageView control properties**
 
 | Property             | Type        | Description                              |
 |--------------------|-----------|----------------------------------------|
 | `ResourceUrl`        | `string`    | The file path of the Image.        |
-| `ImageMap`           | `Map`       | Property map associated with a given image. |
+| `Image`              | `Map`       | Property map associated with a given image. |
 | `PreMultipliedAlpha` | `bool`      | Whether the image is opacity-adjusted<br>If `PreMultipliedAlpha` is `true`, the RGB  components represent the color of the object or pixel, adjusted for its opacity by multiplication. If `false`, the opacity is ignored.  |
 | `PixelArea`          | `Vector4`   | Image subarea defined with relative area  values: x coordinate for the top-left corner, y coordinate for the top-left corner, width, and height. To set the subarea as the whole image area, use `[0.0, 0.0, 1.0, 1.0]`.<br>For example, on a 200 x 200 pixel image, the  value `[0.25, 0.5, 0.5, 0.5]` represents a subarea of   that image with the  following coordinates:<br><br>-   Top left: 50, 100<br>-   Top right: 150, 100<br>-   Bottom left: 50, 200<br>-   Bottom right: 150, 200        |
 | `Border`             | `Rectangle` | Image border, specified  in the following order:  left, right, bottom, and  top. For N-Patch images only.         |
 | `BorderOnly`         | `bool`      | Whether to draw only the borders. For N-Patch  images only.  |
-| `SynchronousLoading` | `bool`      | Whether the image is synchronous. For N-Patch images only.     |
+| `SynchronosLoading`  | `bool`      | Whether the image is synchronos. For N-Patch images only.     |
```

Comment:
I think you could have retained the correct spelling here...

Whether the image is synchronous. For N-Patch images only.